src/main/java/pages/RadioButton.java

```diff
@@ -0,0 +1,27 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+public class RadioButton {
+    private WebDriver driver;
+
+    public RadioButton (WebDriver driver){
+        this.driver = driver;
+    }
+    public void clickYes(){
+        Actions actions = new Actions(driver);
+        WebElement element = driver.findElement(By.id("yesRadio"));
+        actions.click(element).perform();
+    }
+    public void clickImpressive(){
+        Actions actions = new Actions(driver);
```

Comment:
Why do we need to use actions for the click? Why do we have to make an element if it is just going to be used once? These three lines can be written as one:
driver.findElement(By.id("yesRadio")).click